src/hidimstat/utils.py

```diff
@@ -94,14 +187,34 @@ def _ebh_threshold(evals, fdr=0.1):
             selected_index = i
             break
     if selected_index <= n_features:
-        return evals_sorted[selected_index]
+        threshold = evals_sorted[selected_index]
```

Comment:
Why not directly return  `evals_sorted[selected_index]` ?
In Nilearn, this pattern is preferred.